### DISCUSSION TITLE
I am unable to authenticate my app

### DISCUSSION THREAD

**Discussion by poojasap** [osiset/laravel-shopify#902] [Q&A]
Can any one help as i am unable to authenticate the app its throwing me and 404 error 
![image](https://user-images.githubusercontent.com/87488005/125810783-d33e0d86-d893-4b12-b439-b5bc8a1e692f.png)

**Comment by mhmdbdwy** [comment]
please add .htaccess file and put that content into it, it will work

```
Options +FollowSymLinks -Indexes
RewriteEngine On
RewriteCond %{HTTP:Authorization} .
RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
RewriteCond %{REQUEST_FILENAME} !-d
RewriteCond %{REQUEST_FILENAME} !-f
RewriteRule ^ index.php [L]
```

**Reply by mhmdbdwy** [reply]
that .htaccess should be at the main folder
